### DISCUSSION TITLE
Vue Docs update idea

### DISCUSSION THREAD

**Discussion by MichaHuhn** [vuejs/docs#2471] [Ideas]
The Vue Docs suggest to create a copy of the original array before calling `array.reverse()` / `array.sort()`.
However, `array.toReversed()` / `array.toSorted()` would be the better option in this case.
Maybe someone could update this part in the Vue Docs.

[Vue Docs](https://vuejs.org/guide/essentials/list.html#displaying-filtered-sorted-results)
![image](https://github.com/vuejs/docs/assets/69698193/2a31ed7a-1bf7-4722-bf0e-0673252c21b5)

**Comment by brc-dd** [comment]
Browser support of `toReversed` and `toSorted` is very limited (chrome v110+, safari 16+, firefox 115+)

**Reply by MichaHuhn** [reply]
Ahh thank you for this information.
Could you share a link?

**Reply by brc-dd** [reply]
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toReversed#browser_compatibility

**Reply by MichaHuhn** [reply]
Thanks
